lib/asg.go

```diff
@@ -144,8 +136,7 @@ func GetInstanceTypeForAsg(awsSess *session.Session, asgName string) string {
 		return GetInstanceTypeFromLaunchTemplate(awsSess, *asg.LaunchTemplate.LaunchTemplateName)
 	}
 
-	fmt.Println("Unable to determine the ASG instance type. No Launch Template nor Launch Configuration is defined.")
-	os.Exit(1)
+	log.Fatalln("Unable to determine the ASG instance type. No Launch Template nor Launch Configuration is defined.")
 	return ""
```

Comment:
I would think IDE would complain about unreachable code here. (Just a comment, no need to change)

Reply:
Actually, Go requires it. Strange, but true.